packages/client/src/utils/animations.ts

```diff
@@ -0,0 +1,39 @@
+import { gsap } from 'gsap';
+
+/* 
+https://greensock.com/docs/v3/HelperFunctions
```

Comment:
Do we need all that text?

Reply:
I think it's good to keep it as a reference, it comes from this function is built better to keep it than to have to find it all the time right?